### PR TITLE
improving consistency of azure examples/documentation - 2

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
This is to start the process of having consistent naming in examples/documentation.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
multiple

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_resource.py [hunk 1]
```diff
@@ -109,12 +109,12 @@
 EXAMPLES = '''
   - name: Update scaleset info using azure_rm_resource
     azure_rm_resource:
-      resource_group: "{{ resource_group }}"
+      resource_group: myResourceGroup
       provider: compute
       resource_type: virtualmachinescalesets
-      resource_name: "{{ scaleset_name }}"
+      resource_name: myVmss
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_resource.py:115]: Here you are using 'myVmss' but in other places you have `myVmssName` - I'd prefer `myVmss` to match `myResourceGroup`.

File: lib/ansible/modules/cloud/azure/azure_rm_resource_facts.py [hunk 2]
```diff
@@ -69,10 +69,10 @@
 EXAMPLES = '''
   - name: Get scaleset info
     azure_rm_resource_facts:
-      resource_group: "{{ resource_group }}"
+      resource_group: myResourceGroup
       provider: compute
       resource_type: virtualmachinescalesets
-      resource_name: "{{ scaleset_name }}"
+      resource_name: myVmssName
```
Comment (acozine) [inline lib/ansible/modules/cloud/azure/azure_rm_resource_facts.py]:
```suggestion
      resource_name: myVmss
```

File: lib/ansible/modules/cloud/azure/azure_rm_resource.py (hunk 1 above)
Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_resource.py:115]: yeah, fixed.
I will have more prs following, regarding naming conventions.
just subscription ids and resource groups were most obvious

**Comment by acozine** [comment]
Thanks @zikalino!

File: lib/ansible/modules/cloud/azure/azure_rm_resource.py (hunk 1 above)
Comment (acozine) [reply lib/ansible/modules/cloud/azure/azure_rm_resource.py:115]: great - start with the low-hanging fruit for sure

**Pull request merged**